### ISSUE TITLE
Support for probe interface level translation

### ISSUE THREAD

**Issue by mjakovlevs** [ARMmbed/DAPLink#1069]
Could you tell me if there is currently any support for probe interface voltage level translation, for example a possibility for outputting direction control signals for each bidirectional SWD line on the probe MCU GPIO? I am assuming level translation implementation using dedicated ICs with enable and direction control inputs for each data line.
Any suggestions would be very appreciated.

Thank you,
Mihails

**Comment by BobbyCounts** [comment]
Yes, this should be possible. Look at the lpc55xx implementation as an example as the MCU-Link uses bidirectional level translators in the way you describe.

https://github.com/ARMmbed/DAPLink/blob/dbb10a262c2cbcc0db879f0af12219445067ed79/source/hic_hal/nxp/lpc4322/DAP_config.h#L310C1-L328C2

Implementing these functions can allow you to send the appropriate signals to the level shifter direction pins.
